setup.py

```diff
@@ -78,6 +78,8 @@ def get_dir(system_path=None, virtual_path=None):
             'moduleframework-cmd = moduleframework.bashhelper:main',
             'modulelint = moduleframework.modulelint:main',
             'mtf-generator = moduleframework.mtf_generator:main',
+            'mtf-env-set = moduleframework.mtf_environment:mtfenvset',
```

Comment:
It is possible. But I'm not sure if it is generally good idea.
Because:
 * run all *.py files (there may be also not tests, some common python files. It leads to errors when scheduling or when you have there some tests not ready to schedule.
 * it is more complicate found which tests will be scheduled
 * I prefer to have explicit set of tests in production like in makefile TESTS=sanity1.py sanity2.py smoke.py (instead of *.py) it is also more compatible with ansible standard test roles. They want to have explicit test set.
 * also we support also behave, or frameworks what you choose. so that. mtf -> can be bash alias like: alias mtf="avocado run" but it is just trivial shortcut

So yep, I can imagine that mtf could be alias to "avocado run",  but not to avocado run *.py, because it can leads to bad behaviour.